packages/devtools_app/web/devtools_analytics.js

```diff
@@ -44,18 +44,18 @@ function initializeGA() {
     gtag('config', GA_DEVTOOLS_PROPERTY, {
       'custom_map': {
         // Custom dimensions:
-        dimension1: 'user_app',
-        dimension2: 'user_build',
-        dimension3: 'user_platform',
-        dimension4: 'devtools_platform',
-        dimension5: 'devtools_chrome',
-        dimension6: 'devtools_version',
-        dimension7: 'ide_launched',
-        dimension8: 'flutter_client_id',
+        'dimension1': 'user_app',
```

Comment:
in JS, writing dimension1 in this position is equivalent to writing 'dimension1' so this shouldn't change any actual functionality but I agree writing 'dimension1' is clearer.